### ISSUE TITLE
REALLY broken

### ISSUE THREAD

**Issue by WillDev12** [antoinebou12/marketwatch#172]
**Describe the bug**
Refuses to load games, returns HTTP error when getting prices, and much more as I can imagine.

**To Reproduce**
1. Import
2. Copy the following code:

``` python
from marketwatch import MarketWatch
marketwatch = MarketWatch("username", "password")
out1 = marketwatch.get_price("AAPL")
out2 = marketwatch.get_games()

print(out1)
print(out2)
```

**Expected behavior**
Return proper JSON output like it used to

**Screenshots**

[one](https://ibb.co/7zbfT35)
[two](https://ibb.co/yF2xbWX)

**Desktop (please complete the following information):**
 - OS: Chrome
 - Browser: Chrome
 - Version Current

**Additional context**
THIS USED TO WORK, however now it doesn't.

**Comment by antoinebou12** [comment]
They changed the api recently i gonna look into it

**Comment by antoinebou12** [comment]
They added a geo blocker to the api and the web pages. What I can do is make the stock api, watchlist works. I am not sure if i can make the game works like before. I will do more tests

**Comment by WillDev12** [comment]
That's all right, I'll try to figure out some other stuff in the meantime.  Also, one last thing -- There was another problem with the output that returned "Working..." every time a request was made.  Not being too familiar with python, stdout was the closest thing I could get to returning the right values to Javascript, but each time I did there was another object sent with that message.  It'd be pretty helpful I'm sure if you could try to wire out that as well, but I'm in no hurry so take your time.
